src/OpenDebugAD7/AD7DebugSession.cs

```diff
@@ -905,6 +908,7 @@ protected override void HandleInitializeRequestAsync(IRequestResponder<Initializ
                 SupportsSetVariable = true,
                 SupportsFunctionBreakpoints = m_engineConfiguration.FunctionBP,
                 SupportsConditionalBreakpoints = m_engineConfiguration.ConditionalBP,
+                SupportsDataBreakpoints = true,
```

Comment:
>true [](http://example.com/codeflow?start=42&length=4)

You want to define an engine configuration option for this.

Reply:
To expand on this:
1. Add `dataBP` to [cppdbg.ad7Engine.json](https://github.com/microsoft/MIEngine/blob/main/src/OpenDebugAD7/cppdbg.ad7Engine.json)
2. Match it with [EngineConfiguration.cs](https://github.com/microsoft/MIEngine/blob/cb8e84f8f350da178b75f43c5582dae71e60c166/src/DebugEngineHost.VSCode/VSCode/EngineConfiguration.cs)